lib/tasks/dev.rake

```diff
@@ -1,3 +1,138 @@
-desc "Fill the database tables with some sample data"
-task({ :sample_data => :environment }) do
+namespace :dev do
+  desc "drops, creates, migrates, and adds sample data"
+  task reset: [
+         :environment,
+         "db:drop",
+         "db:create",
+         "db:migrate",
+         "dev:sample_data",
+       ]
+
+  desc "adds sample data for development environment"
+
+  task sample_data: [
+         :environment,
+         "dev:add_users",
+         "dev:add_grinders",
+         "dev:add_brewers",
+         "dev:add_coffees",
+       ] do
+    puts "done"
+  end
+
+  task add_users: :environment do
+    puts "add users"
+
+    names = ["jun", "thierry", "ian", "tony", "chase", "jesus", "andre", "derek", "nour", "alejandra", "ainsley", "gustavo"]
+
+    names.each do |name|
+      u = User.create(
```

Comment:
What exactly is the function of u? if not using you should be able to delete it.